buildenv/jenkins/triggerGrinder.groovy

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+String json_path = "../2072_test.json"
```

Comment:
It is ok to use this for test purposes, but this should be checked into the repo. This should be a user-provided file. If not, the default location should be `disabledTestParser/output.json' `